tasks/pages/organizations/[uuid].tsx

```diff
@@ -72,19 +70,21 @@ const WardsPage: NextPage = ({ language }: PropsWithLanguage<WardsPageTranslatio
   }
 
   return (
-    <div className={tw('w-screen h-screen flex flex-col')}>
+    <PageWithHeader
+      leftSide={[(
+        <BreadCrumb
+          key="crumbs"
+          crumbs={[
+            { display: translation.organizations, link: '/organizations' },
+            { display: translation.wards, link: `/organizations/${organizationUUID}` }
+          ]}
+        />
+      )]}
```

Comment:
Maybe we should try to implement a way to skip the usage of the `BreadCrumb` component on the actual pages/views.

One simple idea:
1. Create an additional attribute/parameter passing the current navigation stack
2. Prepend the `BreadCrumb`-component to the internal/private `leftSide` array.

Reply:
This would increase the developer experience (such as readability)!